lib/src/androidTest/java/com/soundcloud/android/crop/CropBuilderTest.java

```diff
@@ -63,6 +63,15 @@ public void testMaxSizeSetAsExtras() {
         assertThat(intent.getIntExtra("max_y", 0)).isEqualTo(300);
     }
 
+    public void testMinSizeSetAsExtras() {
+        builder.withMaxSize(200, 200);
```

Comment:
Shouldn't this be `builder.withMinSize(200, 200)`?

Reply:
good catch let me update that